docs/CMakeLists.txt

```diff
@@ -1,15 +0,0 @@
-
-find_package(Doxygen)
-
-add_custom_target(docs)
-
-if (DOXYGEN_FOUND AND DOXYGEN_DOT_FOUND)
-    configure_file(Doxyfile.in Doxyfile @ONLY)
-    add_custom_target(doxygen
-        COMMAND ${DOXYGEN_EXECUTABLE} Doxyfile
-        COMMENT "Generating Doxygen Documentation")
-    add_dependencies(docs doxygen)
-    install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html/ DESTINATION share/doc/aktualizr OPTIONAL)
-else(DOXYGEN_FOUND AND DOXYGEN_DOT_FOUND)
-    message(WARNING "doxygen + dot not found, skipping")
-endif(DOXYGEN_FOUND AND DOXYGEN_DOT_FOUND)
```

Comment:
CI is complaining because CMake still contains a reference to this directory and expects to find this file there.

Reply:
OK, I was overzealous in deleting - just added this back in.